modelservice/src/main/sql/build-db.sql

```diff
@@ -7,10 +7,11 @@ use aa_model_service;
 
 create table metric
 (
-  id        int unsigned primary key not null auto_increment,
-  m_key     varchar(100) unique not null,
-  hash      varchar(100) unique not null,
-  tags      json
+  id           int unsigned primary key not null auto_increment,
+  m_key        varchar(100) unique not null,
```

Comment:
Use `key` (backticks) instead of m_key please. One of the other keys uses `key` so let's be consistent.